app/controllers/decidim/votings/vote_confirmation/confirmations_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module VoteConfirmation
+      class ConfirmationsController < Decidim::Votings::VoteConfirmation::ApplicationController
+        def confirm
+          voter_id = params[:uid]
+          render json: { result: "ko" } and return unless voter_id
+          vote = Vote.where(voter_identifier: voter_id).first || SimulatedVote.where(voter_identifier: voter_id).first
```

Comment:
I think we should also check that the voting_id matches the Vote's voting voting_id here.

Reply:
ok